src/containers/board-container.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { Light } from '../components/light.js';
+import { Board } from '../components/board.js';
+import { startingBoardOne } from '../components/test-board.js';
+
+export class BoardContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            board: [],
+            numMoves: 0,
+            hasWon: false,
+        };
+        this.board = this.getNewBoard();
+    }
+
+    /**
+     * Decides which tiles start off in the On positon.
+     * If no lights get turned on, the method will call itself again.
+     */
+    getNewBoard = () => {
+        if (this.props.isTest) return startingBoardOne;
```

Comment:
Personally, I don't like to have this kind of conditional test-only behavior represented as logic within a component. An alternative you could explore is just to allow an `initialBoard` to be passed in as a prop and, when present, use that instead of generating a new one.